internal/server/stat/golden/get_stat_set_within_place/epa_facility.json

```diff
@@ -2510,18 +2503,11 @@
           "value": 1980.945,
           "meta_hash": 3902333627
         },
-        "epaGhgrpFacilityId/1010409": {},
         "epaGhgrpFacilityId/1010417": {
           "date": "2019",
           "value": 232640.788,
           "meta_hash": 3902333627
         },
-        "epaGhgrpFacilityId/1010431": {},
-        "epaGhgrpFacilityId/1010443": {},
-        "epaGhgrpFacilityId/1010516": {},
-        "epaGhgrpFacilityId/1010517": {},
-        "epaGhgrpFacilityId/1010523": {},
-        "epaGhgrpFacilityId/1010533": {},
         "epaGhgrpFacilityId/1010540": {
```

Comment:
Is this deletion also related to the empty entity removal, and is this done only for stat set within place or for all endpoints?

If only for this endpoint it could be ok to remove. My worry is if this would require clients to change the way they access data (since they can't assume that the keys would be present, especially for world pop example below).